vscode/src/chat/chat-view/ChatController.ts

```diff
@@ -622,20 +618,30 @@ export class ChatController implements vscode.Disposable, vscode.WebviewViewProv
             if (!model) {
                 throw new Error('No model selected, and no default chat model is available')
             }
+            const { isPublic: repoIsPublic, repoMetadata } = await firstResultFromOperation(
+                publicRepoMetadataIfAllWorkspaceReposArePublic
+            )
 
-            const sharedProperties = {
+            const telemetryProperties = {
                 requestID,
                 chatModel: model,
+                authStatus,
                 source,
                 command,
-                traceId: span.spanContext().traceId,
                 sessionID: this.chatBuilder.sessionID,
-            }
-            await this.recordChatQuestionTelemetryEvent(
-                authStatus,
-                mentions,
-                sharedProperties,
-                inputText
+                repoMetadata,
+                repoIsPublic,
+                traceId: span.spanContext().traceId,
+                promptText: inputText,
```

Comment:
For the collection of transcript data, we'd need to do the same `isDotCom()` check that was happening before, is this happening somewhere else in the code now?

The reason for this is two-fold, first prevents the local PG instance from storing this data and increasing db load and second for additional security around this sensitive data. 

```suggestion
promptText:
                    isDotCom(authStatus) &&
                    (await truncatePromptString(inputText, CHAT_INPUT_TOKEN_BUDGET)),
            },
```